Classes/ReportsConfigurationStatus.php

```diff
@@ -58,9 +58,9 @@ public function getLangModes() {
 			'fail' => array(),
 		);
 		$value = $GLOBALS['LANG']->sL('LLL:EXT:languagevisibility/locallang_db.xml:reports.ok.value');
-		$severity = tx_reports_reports_status_Status::OK;
+		$severity = \TYPO3\CMS\Reports\Status::OK;
 
-		$rootTpls = \TYPO3\CMS\Backend\Utility\BackendUtility::getRecordsByField('sys_template', 'root', '1', '', 'pid');
```

Comment:
Because of Mysql 5.7, it throws error. 

The query (SELECT * FROM sys_template WHERE root='1' AND sys_template.deleted=0 AND (sys_template.t3ver_state <= 0 OR sys_template.t3ver_wsid = 0) GROUP BY pid) won't work because of error "Expression #1 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'gws.sys_template.uid' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by" (line 63)

$tpl requires only pid and uid, previous versions of mysql chose randomly/first uid when you grouped result by pid.